File: CS/WinSolution.Module/DomainObject1.cs
using DevExpress.Xpo;
using DevExpress.Persistent.Base;

namespace WinSolution.Module {
    [DefaultClassOptions]
    public class DomainObject1 : DomainObjectBase {
        public DomainObject1(Session session) : base(session) { }
        private string _propertyName1;
        public string PropertyName1 {
            get { return _propertyName1; }
            set { SetPropertyValue("PropertyName1", ref _propertyName1, value); }
        }
    }
}

File: VB/WinSolution.Module/DomainObject1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Persistent.Base

Namespace WinSolution.Module
	<DefaultClassOptions> _
	Public Class DomainObject1
		Inherits DomainObjectBase
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private _propertyName1 As String
		Public Property PropertyName1() As String
			Get
				Return _propertyName1
			End Get
			Set(ByVal value As String)
				SetPropertyValue("PropertyName1", _propertyName1, value)
			End Set
		End Property
	End Class
End Namespace
